src/features/architect/tradeMachine/TradeReceiptPanel.jsx

```
@@ -0,0 +1,301 @@
+            {receipt.isLegal ? '✓ LEGAL' : '✗ ILLEGAL'}
+          </span>
+          <span className="text-white/40 text-xs">
+            v{receipt.validatorVersion}
+          </span>
+        </div>
+        <div className="flex items-center gap-2">
+          <button
+            onClick={handleCopyReceipt}
+            className="px-2 py-1 bg-[#222] text-white/70 hover:text-white rounded border border-white/20 hover:bg-[#333] text-xs"
+          >
+            {copyFeedback || 'Copy JSON'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)